internal/ingress/controller/nginx.go

```diff
@@ -846,16 +844,19 @@ func configureDynamically(pcfg *ingress.Configuration, port int, isDynamicCertif
 		return err
 	}
 
-	url = fmt.Sprintf("http://localhost:%d/configuration/general", port)
-	err = post(url, &ingress.GeneralConfig{
+	statusCode, _, err := nginx.NewPostStatusRequest("/configuration/general", "application/json", ingress.GeneralConfig{
 		ControllerPodsCount: pcfg.ControllerPodsCount,
 	})
 	if err != nil {
 		return err
 	}
 
+	if statusCode != http.StatusCreated {
+		return fmt.Errorf("unexpected error code: %d", statusCode)
+	}
```

Comment:
I wonder why did we not do this before. Also where are you not doing the same above for backends?

Reply:
check added to backends